From 22ac94ba989cf944efdc8d42fcea03de41239dd3 Mon Sep 17 00:00:00 2001
From: John Zhuge <jzhuge@cloudera.com>
Date: Wed, 17 May 2017 20:12:52 -0700
Subject: [PATCH 2442/2848] HADOOP-14173. Remove unused
 AdlConfKeys#ADL_EVENTS_TRACKING_SOURCE.
 Contributed by John Zhuge.

Change-Id: I5dc6f885816b8834f718874542dfa373458b0333
(cherry picked from commit 5a40bafdaeec693e613aa02e79dbaaccfdab6f60)
---
 .../java/org/apache/hadoop/fs/adl/AdlConfKeys.java |    1 -
 1 file changed, 1 deletion(-)

diff --git a/hadoop-tools/hadoop-azure-datalake/src/main/java/org/apache/hadoop/fs/adl/AdlConfKeys.java b/hadoop-tools/hadoop-azure-datalake/src/main/java/org/apache/hadoop/fs/adl/AdlConfKeys.java
index 7d31103..8fc8e00 100644
--- a/hadoop-tools/hadoop-azure-datalake/src/main/java/org/apache/hadoop/fs/adl/AdlConfKeys.java
+++ b/hadoop-tools/hadoop-azure-datalake/src/main/java/org/apache/hadoop/fs/adl/AdlConfKeys.java
@@ -66,7 +66,6 @@
   static final String ADL_HADOOP_CLIENT_NAME = "hadoop-azure-datalake-";
   static final String ADL_HADOOP_CLIENT_VERSION =
       "2.0.0-SNAPSHOT";
-  static final String ADL_EVENTS_TRACKING_SOURCE = "adl.events.tracking.source";
   static final String ADL_EVENTS_TRACKING_CLUSTERNAME =
       "adl.events.tracking.clustername";
 
-- 
1.7.9.5

